aloe/parser.py

```diff
@@ -240,14 +257,14 @@ def hashes(self):
         """
 
         if not self.table:
-            return []
+            return ()
```

Comment:
Does this constitute an API change for anyone assuming `hashes :: () -> List<...>`